### PR TITLE
Added FAQ

### PR DESCRIPTION
**Pull request by 0n1cOn3**
## Description

NOTE: For the PR to be reviewed it is compulsory to mention the reviewed issue.  
Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes:

- Mention issue using `#`

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] This PR fixes a reviewed issue.(If not create an issue first)
- [x] This PR points to `dev` branch 
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by 0n1cOn3** [comment]
Points to all Packages !

File: README.md [hunk 1]
```diff
@@ -13,10 +13,7 @@
 
 
 > ## Deprecation Warning:
-> **All TBomb versions below v2.0 will no longer work after 14-11-2020.**  
-**All TBomb users need to update to v2.0 ASAP**
-
-**Due to overuse of script, a bunch APIs have been taken offline. It is okay if you do not receive all the messages.**
+> **Due to overuse of script, a bunch APIs have been taken offline. It is okay if you do not receive all the messages.**
```
Comment (t0xic0der) [inline README.md:16]: Please correct this to `Due to the overuse of script, a bunch of APIs have been taken offline. It is okay if you do not receive all the messages.`

File: README.md [hunk 2]
```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
 For Queries: [Telegram Group](https://t.me/TBombChat)  
 [Check Out My YouTube Channel](https://www.youtube.com/c/SpeedXTech)
 
+## FAQ
+
+⁃ Internet Connection poor:
```
Comment (t0xic0der) [inline README.md:161]: Change this to `Poor internet connection`.

File: README.md [hunk 3]
```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
...
+
+```Check your connection and try again. Maybe ping any adress to be sure its not your line.```
```
Comment (t0xic0der) [inline README.md:163]: Address is misspelled. 

Also, rephrase the sentence to `Check your internet connection and try pinging any remote address to confirm`.

File: README.md [hunk 4]
```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
...
+
+⁃ Do you support "X" Country?
+
+```Most Countries are supported for sms and only India for calls. The SMS delivery rate might be different for different countries.```
```
Comment (t0xic0der) [inline README.md:167]: Change `sms` to `SMS` for consistency.

File: README.md [hunk 5]
```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
...
+
+⁃ Can you add support for "X" Country?
+
+```We do what we can, but we cannot promise. Please stay tuned for future support. If you are ready to help then maybe we can do faster.```
```
Comment (t0xic0der) [inline README.md:171]: Requesting rephrasing the sentence.

File: README.md [hunk 6]
```diff
@@ -159,3 +156,52 @@ The package `toilet` cannot be installed yet. But TBomb does still work.
...
+
+⁃ Why is the Limit so low?
```
Comment (t0xic0der) [inline README.md:173]: Change `Limit` to `limit`.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
A general rephrasement of the entire document is needed.

File: README.md (hunk 1 above)
Comment (0n1cOn3) [reply README.md:16]: Welp, how do i am do that? In my Branch?

**Pull request merged**